src/index.hot.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { AppContainer as Hot } from 'react-hot-loader';
+
+import RootRender from '../src';
+
+const render = () => {
```

Comment:
Do you want to try to ask in react-hot-loader's gitter? https://gitter.im/gaearon/react-hot-loader